infra/syntax/Json.cpp

```diff
@@ -848,8 +855,7 @@ namespace infra
 
         tokenizer = current;
 
-        if ((!token.Get<JsonBiggerInt>().Negative() && token.Get<JsonBiggerInt>().Value() <= std::numeric_limits<int32_t>::max())
-            || (token.Get<JsonBiggerInt>().Negative() && token.Get<JsonBiggerInt>().Value() <= static_cast<uint64_t>(-static_cast<int64_t>(std::numeric_limits<int32_t>::min()))))
+        if ((!token.Get<JsonBiggerInt>().Negative() && token.Get<JsonBiggerInt>().Value() <= std::numeric_limits<int32_t>::max()) || (token.Get<JsonBiggerInt>().Negative() && token.Get<JsonBiggerInt>().Value() <= static_cast<uint64_t>(-static_cast<int64_t>(std::numeric_limits<int32_t>::min()))))
```

Comment:
Holy moly, a line of 297 characters! Even I would not make a line so long.

Reply:
Perhaps this needs a // clang-format off

Reply:
I would have expted clang-format to properly align (if set) too long lines

It is definitely capable of doing so.

Reply:
But for that to work you need to set the `ColumnLimit: 0` to some vlaue that makes sense

Reply:
Setting the ColumnLimit to something other than 0 has some side-effects. Setting it is going to be a trade-off.

It means that when you define a class with a single base it will be put on one line. And it formats lambda's differently, this might be a good thing in some cases.

See https://stackoverflow.com/questions/61082996/when-does-clang-format-break-line-of-the-inheritance-list for an example.

Reply:
I would wait for that issue to be resolved before setting a ColumnLimit